How to Find and Use Technical Resources if You’re New to Tech:proposal.md

```diff
@@ -0,0 +1,60 @@
+# Outline - "How to Find Coding Resources if You're New to Tech"
+
+### Step One
+
+**Title:**
+
+How to Find Coding Resources if You're New to Tech
+
+**Draft:**
+Why not dip your toes into the world of coding if you have some extra time on your hands? Whether you're bored during summer or want to learn new skills, learning a bit of coding is a great idea. Knowing how to code is invaluable knowledge, not matter what level of mastery you're at. And as a bonus, it gives you the ability to build and create awesome projects with more flair and creativity. 
+  
+**What is coding?**
```

Comment:
Don't really need the "what is coding" section - I'd assume that if someone checks out your blog they already know about coding and are looking for ways in which they can find better resources. Typically aimed for high school/freshmen college students, who try to jump into coding projects without doing sufficient research.